README.rst

```diff
@@ -1022,6 +1311,70 @@ validating custom parameters of a ``Check`` object.
 This setting allows you to choose whether `config_applied <#configuration-applied>`_ checks should be
 created automatically for newly registered devices. It's enabled by default.
 
+``OPENWISP_MONITORING_AUTO_IPERF``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``False``   |
++--------------+-------------+
+
+This setting allows you to choose whether `iperf <#iperf-1>`_ checks should be
+created automatically for newly registered devices. It's disabled by default.
+
+``OPENWISP_MONITORING_IPERF_CHECK_RSA_KEY_PATH``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------------------------+
+| **type**:    | ``str``                       |
++--------------+-------------------------------+
+| **default**: | ``/tmp/iperf-rsa-public.pem`` |
++--------------+-------------------------------+
+
+This setting allows you to choose RSA public key path (relative to ``/root``)
+for `iperf check running with authentication <#configure-iperf-check-for-authentication>`_.
```

Comment:
I don't understand, isn't this something that depends on a specific server? Why do we have a global setting?

Reply:
Yes it should be defined in [`OPENWISP_IPERF_CHECK_CONFIG`](https://github.com/openwisp/openwisp-monitoring/tree/issue-385/iperf-check#openwisp_monitoring_iperf_check_config)